File: src/main/java/orbapi/MethodCall.java
package orbapi;

import java.util.Arrays;

public class MethodCall{
    private String methodName;
    private Object[] args;

    public MethodCall() {
    }

    MethodCall(String methodName, Object[] args) {
        this.methodName = methodName;
        this.args = args;
    }

    public String getMethodName() {
        return methodName;
    }


    public Object[] getArgs() {
        return args;
    }

    @Override
    public String toString() {
        return "MethodCall{" +
                "methodName='" + methodName + '\'' +
                ", args=" + Arrays.toString(args) +
                '}';
    }
}

File: ObjectRequestBrokerCS/ORB/orbapi/MethodCall.cs
namespace ORB.orbapi
{

	public class MethodCall
	{
		private string _methodName;
		private object[] _args;

		public MethodCall()
		{
		}

		internal MethodCall(string methodName, object[] args)
		{
			_methodName = methodName;
			_args = args;
		}

		public virtual string MethodName
		{
			get
			{
				return _methodName;
			}
		}


		public virtual object[] Args
		{
			get
			{
				return _args;
			}
		}

		public override string ToString()
		{
			return "MethodCall{" +
					"methodName='" + _methodName + '\'' +
					", args=" + _args +
					'}';
		}
	}
}